ansible/roles/common/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+common:
+    bashrc: "{{ lookup('env', 'HOME') }}/.bashrc"
```

Comment:
This will be a problem for people using different shell than Bash (I am Zsh person myself), but I don't think we need to fix this right now, so I'd probably suggest 2 step approach:
1) Document the fact that this playbook currently works only for Bash in a README
2) Add a message to the end of playbook output saying that the env only supports bash

Reply:
It seems that after ansible 1.4 I could use ansilbe_env.HOME to get user's home dir (not sure if that handle zsh but I would expect so). But, this variable is only populated after ansible gathering facts from target machine. On this playbook i disabled gathering facts because it seems ansible needs python installed to get such facts (at least that was what I see on my tests). We could:
```
getent passwd {{ user }} | cut -d: -f6
```
or something like it, into an specific task to get it. Or, again, enforce that python is installed on target machine. Which to be honest should be a fair guess on workstation installs, but a possible problem on lxc/docker images

Reply:
Ahh, sorry I didn't exaplain myself :) `$HOME` should be universal, I meant `.bashrc`

Reply:
oh, ofc! Sorry about that :) OK, in fact we don't need .bashrc anymore. We were using it just to set up snactor env vars, but after latest snactor update it can use a .sanctor.cfg for that. Will update it